docs/eigenlayer/avs-guides/avs-developer-best-practices.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 6
+title: AVS Developer Best Practices
+---
+
+## AVS Developer Best Practices
+
+Note that these best practices should be considered by operators when they’re considering running the AVS software. EigenLabs will not do any checking or scanning of on chain or off chain software.
```

Comment:
rephrase to ` EigenLabs is not responsible for validating on-chain or off-chain software`

Reply:
Good call, done